### ISSUE TITLE
docker executor running in remote vm - need logging

### ISSUE THREAD

**Issue by gilv** [lithops-cloud/lithops#338]
When Docker executor running inside VM, we need to make sure all logs are written into local file and uploaded to COS at the end. Otherwise user has no way to debug code executions inside VM.

**Comment by gilv** [comment]
I see proxy.log in pywren-docker  folder which contains all logs from the execution

**Comment by gilv** [comment]
@JosepSampe this is great #425 . Thanks a lot. Can you explain how it works -  User will just see all the logs on the console and it will include remote logs from VM?

**Comment by JosepSampe** [comment]
Remote VM function logs are synchronized into a local file, you only need to:
`cat /tmp/lithops/functions.log` or ` tail -f /tmp/lithops/functions.log`

/tmp folder can vary depending of the OS, so if you set logging to INFO, you will see a line like this in the executor logs:

`2020-11-03 12:16:34,683 [INFO] lithops.standalone.standalone: View execution logs at /tmp/lithops/functions.log`

**Comment by gilv** [comment]
@JosepSampe EMBL will run multiple jobs in the same Gen2. We tested that each workload in Gen2 will run in it's Docker image...do you think the functions.log will be overwritten? 

I wonder if we want to keep this local file as functions-<job-id>.log

Then users can simply write lithops.log(invocation_id) and get content of this local file

**Comment by JosepSampe** [comment]
Now logs are always appended into the same `functions.log`, they are not overwritten

**Comment by gilv** [comment]
@JosepSampe can we add support  to write logs into `functions-<job-id>.log`

Then users can simply write lithops.log(invocation_id) and get content of this local file?

this will be very usefull

**Comment by JosepSampe** [comment]
@gilv I run some tests with this and maybe create a separate log file per `job-id` is to fine-grained logging. Only tests generate 50 different log files. This could lead to hundred of different files in your local machine.

In IBM CF the command `ibmcloud fn activation poll` shows all finished activations from any job, and it is easy to debug functions. 
Jobs are executed one after another, so what do you think if we generate a different log file per `FunctionExecutor()` instead?

**Comment by gilv** [comment]
@JosepSampe i guess in many cases, like EMBL, it's not a local machine that start Lithops job, but from some VM.
I see a lot of value to retrieve a log based on the job id and this log contains all details, from CF, VM, etc. Also EMBL asked this. If we generate log based on the FunctionExecutor it will be very hard to follow a particular job that perhaps failed.

But i agree there might many logs generated eventually, however it's not only our issue. Many projects generate logs..eventually users just clean them manually

**Comment by JosepSampe** [comment]
> If we generate log based on the FunctionExecutor it will be very hard to follow a particular job that perhaps failed.

In fact if a job fails, the whole execution is stopped, so you will always see failed logs at the end of the `executor-id.log
`

**Comment by gilv** [comment]
@JosepSampe right...but still i don't see issues in many log files..it even good, since it can be later used for some statistics and learning of average executions. And user can get logs based on job id...

**Comment by gilv** [comment]
@JosepSampe are you ok with the above?
